MediaBrowser.Model/Entities/MediaAttachment.cs

```diff
@@ -0,0 +1,50 @@
+namespace MediaBrowser.Model.Entities
+{
+    /// <summary>
+    /// Class MediaAttachment
+    /// </summary>
+    public class MediaAttachment
+    {
+        /// <summary>
+        /// Gets or sets the codec.
+        /// </summary>
+        /// <value>The codec.</value>
+        public string Codec { get; set; }
+
+        /// <summary>
+        /// Gets or sets the codec tag.
+        /// </summary>
+        /// <value>The codec tag.</value>
+        public string CodecTag { get; set; }
+
+        /// <summary>
+        /// Gets or sets the comment.
+        /// </summary>
+        /// <value>The comment.</value>
+        public string Comment { get; set; }
+
+        /// <summary>
+        /// Gets or sets the index.
+        /// </summary>
+        /// <value>The index.</value>
+        public int Index { get; set; }
+
+        /// <summary>
+        /// Gets or sets the filename.
+        /// </summary>
+        /// <value>The filename.</value>
+        public string Filename { get; set; }
+
+        /// <summary>
+        /// Gets or sets the MIME type.
+        /// </summary>
+        /// <value>The MIME type.</value>
+        public string MIMEType { get; set; }
```

Comment:
Ms would probably write it like this
```suggestion
        public string MimeType { get; set; }
```